### ISSUE TITLE
Incorrect multiline error highlight

### ISSUE THREAD

**Issue by RikardLegge** [SublimeLinter/SublimeLinter-flow#11]
When the linter tries to highlight a multiline error it fails and only marks the first column, including all characters after the error. 

Example:

```javascript
/* @flow */

module.exports.invalid_export = class test{ // Multilne error. (This whole line gets maked as an error, not only the error itself)
	num:number; // Not highlighted
	str:string; // Not highlighted
} // Not highlighted
```

The error from flow looks like:
```
.../test_required.js:3:33,6:1: 
not (sup)ported
```

**Comment by RikardLegge** [comment]
See #9 regarding related problems and proposed solution.

**Comment by ckaznocha** [comment]
Closed by PR #21
